### ISSUE TITLE
Integrate EF Core diagnostics information into ApplicationInsights

### ISSUE THREAD

**Issue by divega** [aspnet/EntityFramework#8272]
We can either extend [Microsoft Application Insights for ASP.NET Core applications](https://github.com/Microsoft/ApplicationInsights-aspnetcore) or create our own package that integrate EF Core's DiagnosticSource (or possibly logging) events into Application Insights monitoring.

**Comment by divega** [comment]
@anpete @ajcvickers I believe one of you is going to start working on this soon. Feel free to self-assign.

**Comment by ajcvickers** [comment]
@anpete @divega How should we track this given that it isn't directly related to any EF milestone.

**Comment by anpete** [comment]
@ajcvickers I think it's fine to just say we want to do the work in 2.0 time frame and track it accordingly.

**Comment by ajcvickers** [comment]
Since this isn't directly tired to 2.0 and we will still be working on this post 2.0, moving it out of 2.0 for now.

**Comment by divega** [comment]
Fair enough. Adding [area-eternal]. I think could choose to track in a milestone that fits the timeframe in which we want this completed.

**Comment by ajcvickers** [comment]
Marking for re-triage to decide howto track any future work here when @divega is back.

**Comment by ajcvickers** [comment]
Putting this on the backlog to consider doing more EF Core specific integration in the future. We have done the SQL integration that we planned for 2.1.
